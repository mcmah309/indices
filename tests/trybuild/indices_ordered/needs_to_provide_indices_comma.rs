use indices::indices_ordered;

fn main() {
    let mut data: [i32; 0] = [];
    let slice = data.as_mut_slice();
    let result = indices_ordered!(slice,);
}