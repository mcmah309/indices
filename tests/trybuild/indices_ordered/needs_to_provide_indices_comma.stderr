error: unexpected end of macro invocation
 --> tests/trybuild/indices_ordered/needs_to_provide_indices_comma.rs:6:41
  |
6 |     let result = indices_ordered!(slice,);
  |                                         ^ missing tokens in macro arguments
  |
note: while trying to match meta-variable `$index1:expr`
 --> src/lib.rs
  |
  |     ($slice:expr, $index1:expr) => {{
  |                   ^^^^^^^^^^^^
